app/views/public_recipe/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1 class='recipe-name'>Public Recipes</h1>
+
+<% @public_recipes.each do |recipe| %>
+  <% sum = 0  %>
+  <% recipe.food_recipes.each {|food| sum += food.quantity * food.food.price}%>
+    <div class='public-flex'>
+      <div>
+        <p class='recipe-name'>
+        <%= recipe.name %>
+        </p>
+        <p>
+        <strong class='sub-title'>By:</strong>
+        <%= content_tag(:span, recipe.user.name, class: "sub-title") %>
+        </p>
+      </div>
+      <div class='pos'>
+        <p>
+        <strong class='sub-title '>Total food Item:</strong>
+        <%= content_tag(:span, recipe.food_recipes.count, class: "sub-title") %>
+        </p>
+        <p>
+        <strong class='sub-title'>Total Price:$</strong>
+        <%= content_tag(:span, sum, class: "sub-title") %>
+        </p>
+      </div>
+      
+    </div>
+  <% end %>
```

Comment:
- [ ] Please make sure that each of these `Recipe` cards leads to their respective details page. According to the requirements:

![image](https://github.com/gsmalik030/recipe_app/assets/77069011/2da2cbe9-9639-4c2e-9452-2110b220678b)

Consider fixing it! 🤓 🔧